Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -109,13 +109,14 @@ def get_indicators(self, params):
         )
         return response
 
-    def fetch_indicators(self, limit: Optional[int], offset: Optional[int] = 0, fetch_command=False) -> list:
+    def fetch_indicators(self, limit: Optional[int], offset: Optional[int] = 0, fetch_command=False, use_last_run=False) -> list:
         """ Get indicators from CrowdStrike API
 
         Args:
             limit(int): number of indicators to return
             offset: indicators offset
             fetch_command: In order not to update last_run time if it is not fetch command
+            use_last_run: Wether to use the last_run when calling crowdstrike-indicators-list command.
```

Comment:
I think instead of making this field a boolean, it would be easier for us to debug if we can give it a timestamp.
Eg. use_last_run_timestamp = 10999... so we can have more control over the debugging

Reply:
Oh, I thought we want it to be the same as the one used in the fetch indicators..
I'll change it